src/index.js

```diff
@@ -84,7 +75,7 @@ function routeFromLink(node) {
 		target = node.getAttribute('target');
 
 	// ignore links with targets and non-path URLs
-	if (!href || !href.match(/^\//g) || (target && !target.match(/^_?self$/i))) return;
+	if (!/^\//.test(href) || (target && !target.match(/^_?self$/i))) return;
```

Comment:
Good call.  While we're here, might be worth extending this check to also ignore `javascript:` prefixed `href` values.